packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/utils/helpers/getHighestPriorityConcern.ts

```diff
@@ -3,7 +3,7 @@ import { ProviderVirtualMachine } from '@kubev2v/types';
 type ConcernCategory = 'Critical' | 'Warning' | 'Information';
 
 export const getHighestPriorityConcern = (vm: ProviderVirtualMachine): ConcernCategory => {
```

Comment:
this method is scheduled to be replaced, because we don't want to search by "highest" 

does it makes sens to replace this method in this PR ?

Reply:
we are not ready yet to remove the method and the code won't compile otherwise.

Reply:
ok, works for me